templates/quickstart-jira-dc.template.yaml

```diff
@@ -993,6 +953,22 @@ Resources:
                 'Fn::Join':
                   - "\n"
                   -
+                    - "ATL_PRODUCT_FAMILY=jira"
+                    - "ATL_DB_DRIVER=org.postgresql.Driver"
+                    - "ATL_JDBC_DB_NAME=jira"
+                    - "ATL_JDBC_USER=atljira"
+                    - "ATL_APP_DATA_MOUNT_ENABLED=false"
+                    - "ATL_ENABLED_PRODUCTS=Jira"
+                    - "ATL_ENABLED_SHARED_HOMES="
+                    - "ATL_NGINX_ENABLED=false"
+                    - "ATL_POSTGRES_ENABLED=false"
+                    - "ATL_RELEASE_S3_BUCKET=atlassian-software"
+                    - "ATL_RELEASE_S3_PATH=releases"
+                    - "ATL_SSL_SELF_CERT_ENABLED=false"
+                    - ""
+                    - !Sub ["ATL_PRODUCT_EDITION=${Edition}", Edition: !Ref "JiraProduct"]
```

Comment:
We should stick with one quoting pattern when using `!Ref`, for variables I don't think we should quote them.

Reply:
Fair enough, I'll add a task to the cleanup epic.